api/services/FootprintService.js

```diff
@@ -181,20 +180,33 @@ module.exports = class FootprintService extends Service {
    * @return Promise
    */
   createAssociation(parentModelName, parentId, childAttributeName, values, options) {
-    /*
-     const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
-     const childAttribute = parentModel.associations[childAttributeName]
-     const childModelName = childAttribute.target.name
-
-     values[childAttribute.foreignKey] = parentId
+    const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
+    if (!parentModel) {
+      return Promise.reject(new ModelError('E_NOT_FOUND', `${parentModelName} can't be found`))
+    }
+    const association = parentModel.associations[childAttributeName]
+    if (!association) {
+      return Promise.reject(new ModelError('E_NOT_FOUND', `${parentModelName}'s association ${childAttributeName} can't be found`))
+    }
+    const childModelName = association.target.name
 
-     return this.create(childModelName, values, options)
-     */
-    return Promise.reject('trailpack-sequelize does not have createAssociation support yet. Sorry')
+    if (association.foreignKeyField) {
+      values[association.foreignKeyField] = parentId
+    }
+    return this.create(childModelName, values, options).then(child => {
+      if (association.as) {
+        return this.update(parentModelName, parentId, {
```

Comment:
If this update fails the database will be in a strange state :/ can this be solved by using a transaction for both `create` and `update` who rollback when problem ?